drivers/eeprom/Kconfig.eeprom_emu

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 Laczen
+# SPDX-License-Identifier: Apache-2.0
+
+config EEPROM_EMULATOR
+	bool "Emulated EEPROM driver"
+	select FLASH
+	help
+	  Enable emulated (on flash) EEPROM support.
```

Comment:
This needs a lot more detail. See your commit message for ideas

Reply:
done